src/components/Button/Button.js

```diff
@@ -2,14 +2,35 @@ import { html } from '../../shared';
 import styles from './Button.module.css';
 
 /**
- * @param {{onClick: (e: MouseEvent) => void, text: string}} props
+ * @param {{ text: string, onClick: (e: MouseEvent) => void, variant?: 'normal' | 'outlined' | 'nextQuestion' | 'gameMode', disabled?: boolean }} props
+ * * @param  {...string} id
  * @returns {HTMLButtonElement}
  */
-function Button({ onClick, text }) {
-  const btn = html`<button class="${styles.btn}" type="button">${text}</button>`;
-  btn.addEventListener('click', onClick);
+function Button({ text, onClick, variant = 'normal', disabled = false }, ...id) {
+  const classNamesForVariant = {
+    normal: styles.ButtonNormal,
+    outlined: styles.ButtonOutlined,
+    nextQuestion: styles.ButtonNextQuestion,
+    gameMode: styles.ButtonGameType,
+  };
+  let button;
+  if (variant === 'nextQuestion') {
+    button = html`<button class="${classNamesForVariant[variant]}" type="button" ${disabled ? ' disabled' : ''}>
```

Comment:
Czy zapisu ${disabled ? ' disabled' : ''} nie można uprościć do ${disabled && ' disabled' }?

Reply:
Nie można, bo to jest w stringu, więc dostaniesz 'false' albo 'disabled'